### PR TITLE
Add new future resize editor & Preview Panes by dragging.(#1977)

### PR DESCRIPTION
**Pull request by zhouxiaoxiaoxujian**
You can now dragging the border between Editor & Preview to left/right

### CODE REVIEW

File: browser/components/CodeEditor.js [hunk 1]
```diff
@@ -499,6 +499,7 @@ export default class CodeEditor extends React.Component {
     fontFamily = _.isString(fontFamily) && fontFamily.length > 0
       ? [fontFamily].concat(defaultEditorFontFamily)
       : defaultEditorFontFamily
+    let width = this.props.width
```
Comment (sosukesuzuki) [inline browser/components/CodeEditor.js]: please use `const` instead of `let`.

File: browser/components/MarkdownSplitEditor.styl [hunk 2]
```diff
@@ -3,7 +3,16 @@
...
+    top -2px
+    width 0
+    z-index 0
+  .slider-hitbox
+    absolute top bottom left right
+    width 7px
+    left -3px
+    z-index 10
+    cursor col-resize
+
```
Comment (sosukesuzuki) [inline browser/components/MarkdownSplitEditor.styl]: please remove unnecessary lines!

**Comment by zhouxiaoxiaoxujian** [comment]
Fixed review suggestions. Please check

File: browser/components/MarkdownSplitEditor.styl [hunk 3]
```diff
@@ -13,6 +13,4 @@
     width 7px
     left -3px
     z-index 10
-    cursor col-resize
-
-
+    cursor col-resize
```
Comment (sosukesuzuki) [inline browser/components/MarkdownSplitEditor.styl]: @zhouxiaoxiaoxujian 
please add new line at end of the file!

**Pull request merged**